documentation/docs/reference/software/interfaces/mqtt.md

```diff
@@ -59,171 +109,390 @@ Control of the focus stage. The message is a JSON object, speed is optional:
+| `"Busy"`                            | The camera is currently busy and cannot update settings.                                                  |
 
-State of the imager. It's a JSON object with:
+- **JSON message to stop the imager**:
 
 ```json
 {
-  "status": "Started",
-  "time_left": 25
+  "action": "stop"
 }
 ```
 
-Status is one of `Started`, `Ready`, `Completed` or `12_11_15_0.1.jpg has been imaged`.
+**Authorized values for `stop` action:**
 
-- Publish only
+| Field    | Type   | Accepted Values |
+|----------|--------|-----------------|
+| `action` | string | "stop"          |
 
-#### `status/segmenter`
 
-Status of the segmentation. It's a JSON object with:
+**Status/Error Messages for `stop` action:**
 
-```json
-{
-  "status": "Started"
-}
-```
+| Status/Error message                                 | Description                                                                                                  |
+|---------------------------------------------------|-----------------------------------------------------------------------------------------------------------|
+| `"Interrupted"`                     | The image capture process was stopped successfully.                                                       |
+| `"Busy"`                            | The camera is currently busy and cannot stop the operation.                                               |
 
-`status` is one of `Started`, `Done`, `Interrupted`, `Busy`, `Ready` or `Dead`.
 
-- Publish only
 
-#### `status/segmenter/object_id`
+### `segmenter`
 
-```json
-{
-  "object_id": "13449"
-}
-```
+This topic controls the segmentation process. 
 
-#### `status/segmenter/metric`
+- **JSON message to start `segmentation`**:
+
+The segmentation process analyzes the images stored in the specified path, optionally exporting the results in an ecotaxa-compatible format. The `force`, `recursive`, `ecotaxa`, and `keep` options provide control over how segmentation is performed and managed. 
 
 ```json
 {
-  "name": "01_13_28_232066_0",
-  "metadata": {
-      "label": 0, "width": 29, "height": 80, ....
+  "action": "segment",
+  "path": "/path/to/segment",
+  "settings": {
+    "force": False,
+    "recursive": True,
+    "ecotaxa": True,
+    "keep": True
+  }
 }
 ```
+The `action` element is the only element required. If no `path` is supplied, the whole images repository is segmented recursively (this is very long!).
+
+| Parameter     | Type   | Accepted Values      | Description                                            |
+|---------------|--------|----------------------|--------------------------------------------------------|
+| `path`        | string | path/to/directory    | Path to the directory to segment.                      |
+| `force`       | bool   | true, false          | Force re-segmentation even if previously done. It will overcome the presence of the file `done` that prevents resegmenting a folder already segmented. |
```

Comment:
For nested parameters (e.g. `force`, `recursive`, etc.), I would suggest providing a fully-qualified path name (e.g. `settings`.`force`, `settings`.`recursive`, etc.)